src/app/components/playlist/playlist.rs

```diff
@@ -14,6 +14,14 @@ pub trait PlaylistModel {
 
     fn current_song_id(&self) -> Option<String>;
 
+    fn playlist_song_ids(&self) -> Option<Vec<String>> {
+        None
+    }
+
+    fn playlist_is_playing(&self) -> bool {
+        false
+    }
+
```

Comment:
I get what you were trying to do -- but we don´t need these in the trait. code expecting a PlaylistModel will not need these methods